configure

```diff
@@ -73,6 +73,7 @@ find_pytest()
 }
 
 PYTEST=${PYTEST:-`find_pytest`}
+$PYTEST --version 2>&1 | grep -q "version 3" || unset PYTEST
```

Comment:
We should NOT override if they set PYTEST manually, though.  I think this logic needs to be in find_pytest?